server/storage/backend/backend.go

```diff
@@ -300,11 +304,33 @@ func (b *backend) ConcurrentReadTx() ReadTx {
 	// concurrentReadTx is not supposed to write to its txReadBuffer
 	return &concurrentReadTx{
 		baseReadTx: baseReadTx{
-			buf:     *buf,
-			txMu:    b.readTx.txMu,
-			tx:      b.readTx.tx,
-			buckets: b.readTx.buckets,
-			txWg:    b.readTx.txWg,
+			buf:          buf,
+			bufferCopied: true,
+			mu:           new(sync.RWMutex),
```

Comment:
I am wondering if we really use mu in this case? if not, we can just set it to nil and remove bufferCopied.

Reply:
We can not remove bufferCopied, because function UnsafeRange can be used to search meta data,such as:
```
tx := be.ReadTx()
tx.Lock()
defer tx.Unlock()
assert.Nil(t, UnsafeConfStateFromBackend(lg, tx))
```
In such case, mu is not nil, but we can not lock it again!

So we need a member to know it is used only for buff nocopy.